testsuite/features/core_proxy_branch_network.feature

```diff
@@ -113,11 +113,10 @@ Feature: Setup SUSE Manager for Retail branch network
     Then I should see a "Formula saved!" text
 
 @proxy
-@private_net
-  Scenario: Enable avahi on the branch server
+ @private_net
+  Scenario: Enable repositories for installing branch services
```

Comment:
This is not what this is doing anymore.

I suggest:
```cucumber
Scenario: Install expect package that is needed for testing Retail
```

Reply:
I don't know why expect is needed, but I need repositories enabled for next step when highstate is applied.

Reply:
Ah, so we need both. I thought we needed the repositories to install SUSEfirewall2 and expect.

OK, forget about expect and keep that title (or put it in a separate scenario).